### PR TITLE
Table View: Column 1 should use triangle overlay bullet.

### PR DESCRIPTION
**Pull request by victoray**
#339

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -2351,21 +2351,26 @@ code, .code {
 /* column 1 */
 .table-view > .children > .child > .thought-container {
   display: table-cell;
-  text-align: right;
+  /*text-align: right;*/
```
Comment (raineorshine) [inline src/App.css]: > Pull Requests that solve the given issue but introduce a regression (that is, break or remove functionality) are not considered solutions and will not be accepted. Preserving existing functionality is an implicit requirement.

https://github.com/cybersemics/em/blob/dev/CONTRIBUTING.md#regressions

File: src/App.css [hunk 2]
```diff
@@ -2378,10 +2383,13 @@ code, .code {
...
 
 .table-view > .children > .child > .thought-container > .thought > .bullet-cursor-overlay {
-  top: -15px;
-  left: 4px;
+  top: 2px !important;
+  left: 4px !important;
   margin-left: 0;
 }
+.table-view > .children > .child:first-child > .thought-container > .thought > .bullet-cursor-overlay {
+  top: 10px !important;
```
Comment (raineorshine) [inline src/App.css]: `!important` is often a bad practice. Can you explain why it was necessary here? Is there a cleaner way to do this?

Comment (victoray) [reply src/App.css]: I will check and see if I can fix it without the `!important`

File: src/App.css (hunk 1 above)
Comment (victoray) [reply src/App.css]: with the `text-align: right` the cursor will be too far from the text, I will try and explore other options though and see how they will work.

Comment (raineorshine) [reply src/App.css]: Yes, you will need to find a solution that still allows the text to be right-aligned.

**Comment by raineorshine** [comment]
Fixed in 6abc70c

**Pull request closed without merging**